git_pile/git_pile.py

```diff
@@ -979,7 +979,7 @@ def cmd_am(args):
         return 1
 
     info("Entering '%s' directory" % config.dir)
-    gitdir = git_worktree_get_git_dir()
+    gitdir = git_worktree_get_git_dir(config.dir)
     if op.isdir(op.join(gitdir, "rebase-apply")):
         fatal("Already on an am or rebase operation", file=sys.stderr)
```

Comment:
Should this error messages also be updated to clarify which worktree has the pending operation?  Otherwise I can see this being a bit confusing to the user.

Reply:
yeah. But I will do it in #69